lib/google_analytics/analytics_export_service.rb

```diff
@@ -18,11 +18,12 @@ def build
 
     def export_bad_links(data)
       begin
+        response = ''
         Tempfile.create('bad_links.csv', Dir.pwd) do |file|
           file.write(data)
-          response = service.upload_data(ENV['GA_EXPORT_ACCOUNT_ID'],
-                                         ENV['GA_EXPORT_CUSTOM_DATA_IMPORT_SOURCE_ID'],
-                                         ENV['GA_EXPORT_TRACKER_ID'],
+          response = service.upload_data(ENV['GOOGLE_EXPORT_ACCOUNT_ID'],
+                                         ENV['GOOGLE_EXPORT_CUSTOM_DATA_IMPORT_SOURCE_ID'],
+                                         ENV['GOOGLE_EXPORT_TRACKER_ID'],
                                          fields: "accountId,customDataSourceId,errors,id,kind,status,uploadTime",
```

Comment:
This is another git hygiene nitpick, but this change seems unrelated to the rest of this commit, and could happily live in the commit where you introduce this code: e6362afe5100e286606cf6455eea2cf4b51579b9.